package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ValorToken",
-  "version": "1.0.0",
+  "version": "0.1.1",
```

Comment:
```suggestion
  "version": "1.0.1",
```

Reply:
It's ok, we are not changing the interface of the contract so it's good to advance only the minor version, the previous value was there by mistake